install_config/router/default_haproxy_router.adoc

```diff
@@ -1491,22 +1491,33 @@ for consumption by external metrics collection and aggregation systems (e.g. Pro
 Metrics are also available dirctly from the
 link:https://cbonte.github.io/haproxy-dconv/1.5/configuration.html#9[HAProxy router] in its own CSV format.
 
-When you create a router, as below,
+When you create a router using the following command, metrics are automatically available in Prometheus format on the stats-port, default 1936. 
+
 ====
 ----
 $ oc adm router --service-account=router
 ----
 ====
-metrics are automatically available in Prometheus format on the stats-port, default 1936. To suppress metrics collection,
+
+* To get a CSV text file that contains Prometheus format statistics with *metrics* exposed:
++
+Launch the stats window use the following URL in a browser:
```

Comment:
This isn't going to work since localhost from a browser is likely the machine they are on.  Really they need to tell prometheus to gather from the host:1936 with the appropriate credentials.

Reply:
@knobunc The customer is looking for:
_Please clearly specify both the url to get the metrics and the actual file time returned, txt or html_

If I read our docs right, the URL is the STATS_PORT address set in the router env variables and the user can `curl <user>:<password>@<router_IP>:<STATS_PORT>/metrics`. 
1. `oc get po router-1-6tuk1  --template={{.status.hostIP}}` for router IP
2. `oc describe po router-1-6tuk1` to get the admin name/password?